xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/CategoriesPanel.xml

```diff
@@ -128,7 +128,7 @@
 #panelheader($services.localization.render('xe.blog.categories.paneltitle'))
 {{html wiki="true"}}
 #getCallingDocument($callingDocument)
-* (% class="blog-category-level" %)((( [[#toolImage('bullet_feed')&gt;&gt;Blog.BlogRss||queryString="xpage=plain" title="RSS"]] &lt;span class="wikilink"&gt;&lt;a href="$xwiki.getURL('Blog.WebHome')"&gt;$services.localization.render('blog.categories.all')&lt;/a&gt;&lt;/span&gt;)))
+* (% class="blog-category-level" %)((( [[#toolImage('feed')&gt;&gt;Blog.BlogRss||queryString="xpage=plain" title="RSS"]] &lt;span class="wikilink"&gt;&lt;a href="$xwiki.getURL('Blog.WebHome')"&gt;$services.localization.render('blog.categories.all')&lt;/a&gt;&lt;/span&gt;)))
```

Comment:
Not "feed". "rss". Pleas look at http://playground.xwiki.org/xwiki/bin/view/IconThemesCode/IconPickerMacro and click on the field 1. You will see the list of available icons.